react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -525,11 +547,19 @@ public void onTextChanged(CharSequence s, int start, int before, int count) {
 
         @Override
         public void afterTextChanged(Editable s) {
-            if (!mIsSettingTextFromJS && mListeners != null) {
+            if (shouldRedirectTextChangeCalls()) {
                 for (TextWatcher listener : mListeners) {
                     listener.afterTextChanged(s);
                 }
             }
         }
     }
+
+    private boolean shouldRedirectTextChangeCalls() {
```

Comment:
Nit: what do you think about naming this method as `shouldDelegateTextChangeCalls` to match the class' name?

Reply:
Yup, I agree.